neevaco/neeva-ios
# Quick Fix for Recipe Ratings Crash

We add a check on client side to prevent the case where the normalized ratings it 0 due to incorrect maxRatings value returned from the API.

## Checklists

### How was this tested?

### Manual test cases
Tested on website to make sure it doesn't crash anymore. Also tested on other websites to make sure it still works.

### Screenshots and screen recordings
![Simulator Screen Shot - iPhone 13 - 2022-03-09 at 14 49 49](https://user-images.githubusercontent.com/44557675/157522390-dffd280d-f95c-429f-a803-26748888602b.png)

## Issues addressed
resolves #3023
